benchmark/prov_collectors.py

```diff
@@ -41,7 +42,29 @@ def __str__(self) -> str:
         return self.name
 
     def count(self, log: Path, exe: Path) -> tuple[ProvOperation, ...]:
-        return ()
+
+        parsed_ops = parse_probe_log(log)
+        for pid, process in parsed_ops.processes.items():
+            print(pid)
+            for exid, exec_epoch in process.exec_epochs.items():
+                print(pid, exid)
+                for tid, thread in exec_epoch.threads.items():
+                    print(pid, exid, tid)
+                    for op_no, op in enumerate(thread.ops):
+                        print(pid, exid, tid, op_no, op.data)
+                    print()
+
+        operations = [
+            ProvOperation(
+                type=op.type,
+                target0=op.target0,
+                target1=op.target1,
+                args=op.args
+            )
```

Comment:
We will switch on the type of `op.data`. All of our ops so far have zero or one targets, the args can be left blank.